# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gconf110/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- GConf-1.0.9/backends/bdb.h.orig	2005-09-23 13:43:49.652304995 +0200
+++ GConf-1.0.9/backends/bdb.h	2005-09-23 13:45:02.675975860 +0200
@@ -155,10 +155,8 @@
 
 extern DBT *temp_string_key (const char *key);
 extern DBT *temp_int_key (int akey);
-extern guint32 get_dir_id (BDB_Store * bdb, const char *dir);
 extern void add_key (BDB_Store * bdb, const char *dir, const char *keypath);
 extern void bdb_set_sysname (const char *name);
-extern guint32 get_or_create_dir (BDB_Store * bdb, const char *dir);
 
 int bdb_create (BDB_Store * bdb, const char *dir);
 int bdb_open (BDB_Store * bdb, const char *dir, int flags);
